packages/jsii/test/negatives/neg.enum-name.2.ts

```diff
@@ -1,4 +1,4 @@
-///!MATCH_ERROR: Type names must use PascalCase: My_Enum
+///!MATCH_ERROR: Enum names must use TRUMP_CASE: My_Enum
```

Comment:
Nope! Just the member names should be TRUMP_CASE

Reply:
I think my Java disposition's are showing - enum names vs enum values. I'll get it updated!